### DISCUSSION TITLE
Update user

### DISCUSSION THREAD

**Discussion by vahidtakro** [nextauthjs/next-auth#2924] [Help]
### Question 💬

hello, thanks for the package.
I could make my sign up and sign in pages, now I am working on user setting page, is it possible to do that?
so user should be able to change his name, email, etc.
if yes, how can I do that?

my data are stored with Prisma to database,
should I create an API, if yes, how?

### How to reproduce ☕️

-

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment]
`next-auth` currently doesn't give you primitives for user updates. there is a related discussion at #2269
